msteams-platform/concepts/design/enable-app-customization.md

```diff
@@ -1,41 +1,41 @@
 
 * Changing the app's accent color to match an org's brand.
 * Updating the app name from *Contoso* to *Contoso Agent*, which is the name users in the org will see.
 (Note: Users adding a connector to a chat or a channel will still see the original app name, *Contoso*.)
+* Changing the description to include language and guidance that is better understood by your organization's end-users.
+* Changing logo and logo outline to inspire confidence to the end-users to use the app since it seems provided by their organization.
 
-You can enable this feature by defining the app properties that your customers can customize in the [`configurableProperties` section in the Teams app manifest](/microsoftteams/platform/resources/schema/manifest-schema#configurableproperties), starting with version 1.11. That can be done in the [Developer Portal for Teams](https://dev.teams.microsoft.com/home) if you've chosen to use the Developer Portal to edit the manifest of your app.
+You can enable this feature by defining the app properties in the [`configurableProperties` section in the app manifest file](/microsoftteams/platform/resources/schema/manifest-schema#configurableproperties). You can define the property using [Developer Portal for Teams](https://dev.teams.microsoft.com/home) or do it manually. You can choose to  use the Developer Portal to edit the manifest of your app or you can edit it manually.
+
+:::image type="content" source="../../assets/images/dev-portal-app-customization-settings.png" alt-text="Screenshot shows the option in Teams Developer Portal to enable app customization feature and select specific metadata fields that developers allow admins to customize.":::
 
 > [!IMPORTANT]
 > You can't test this feature during development. App customization isn't supported when sideloading or publishing to an org's app catalog.
 
-### User considerations
-
-Provide guidelines for customers (specifically Teams admins) who want to customize your app. For more information, see [customize apps in Teams](/MicrosoftTeams/customize-apps).
-
 ## Block apps by default for users until an admin approves
 
-To enhance Teams app experience, you can hide an app from users by default until admin allows to unhide the app. For example, Contoso Electronics has created a help desk app for Teams. To enable appropriate functioning of the app, Contoso Electronics’ wants the customers to first set up specific properties of the app. The app is hidden by default and is available to users only after the admin allows it.
+To enhance Teams app experience, you can block an app from users until an admin approves the app. For example, Contoso Electronics has created a help desk app for Teams. To enable appropriate functioning of the app, Contoso Electronics’ wants the customers to first set up specific properties of the app. The app is blocked by default and is available to users only after the admin allows it. The functionality is called blocked by publisher.
```

Comment:
Is this information validated by our feature crew?

Can we please get an approval from them. Rest looks fine.